### PR TITLE
unbound: fix DOCTYPE in plist file

### PR DESCRIPTION
**Pull request by zbeekman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by zbeekman** [comment]
Thanks to @maxim-belkin for spotting this.

File: Formula/unbound.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@ def post_install
 
   def plist; <<~EOS
     <?xml version="1.0" encoding="UTF-8"?>
-    <!DOCTYPE plist PUBLIC "-/Apple/DTD PLIST 1.0/EN" "http:/www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```
Comment (retokromer) [inline Formula/unbound.rb]: Why not the secure `https://www.apple.com/DTDs/PropertyList-1.0.dtd`?

**Comment by fxcoudert** [comment]
While we're at it:

```
dnscrypt-proxy.rb:    <!DOCTYPE plist PUBLIC "-/Apple/DTD PLIST 1.0/EN" "http:/www.apple.com/DTDs/PropertyList-1.0.dtd">
stubby.rb:    <!DOCTYPE plist PUBLIC "-/Apple/DTD PLIST 1.0/EN" "http:/www.apple.com/DTDs/PropertyList-1.0.dtd">
unbound.rb:    <!DOCTYPE plist PUBLIC "-/Apple/DTD PLIST 1.0/EN" "http:/www.apple.com/DTDs/PropertyList-1.0.dtd">
```

**Comment by zbeekman** [comment]
ah yes... I thought it should be just http, but the url worked when I tried it in a browser, and didn't know for sure.

**Comment by zbeekman** [comment]
@fxcoudert Reverted https change and fixed the other two formulae

**Pull request merged**